src/main/java/com/googlecode/lanterna/graphics/DefaultMutableThemeStyle.java

```diff
@@ -55,7 +55,7 @@ public DefaultMutableThemeStyle(ThemeStyle themeStyleToCopy) {
      * @param sgrs Modifiers to apply to the text with this style
      */
     public DefaultMutableThemeStyle(TextColor foreground, TextColor background, SGR... sgrs) {
-        this(foreground, background, EnumSet.copyOf(Arrays.asList(sgrs)));
+        this(foreground, background, sgrs.length > 0 ? EnumSet.copyOf(Arrays.asList(sgrs)) : EnumSet.allOf(SGR.class));
```

Comment:
shouldn't that be "noneOf", rather than "allOf"?

Reply:
Yes you're right! Don't know how I made that mistake hahah. Fixing it now.

Reply:
Fixed :)